wgpu/src/lib.rs

```diff
@@ -1999,13 +2029,24 @@ impl Instance {
     }
 
     /// Generates memory report.
-    #[cfg(not(webgpu))]
+    ///
+    /// Like `try_generate_report` but panics if the active backend is WebGPU.
```

Comment:
Couldn't this just be optional?

Reply:
tbh I have the two versions mostly I found it too tedious to patch all existing `generate_report` calls. Can revisit

Reply:
remove the extra report method again, wasn't _that_ bad